assets/src/apps/delivery/Delivery.tsx

```diff
@@ -24,7 +24,7 @@ export interface DeliveryProps {
   resourceAttemptGuid: string;
   activityGuidMapping: any;
   previewMode?: boolean;
-  enableHistory?: boolean;
+  allowNavigation?: boolean;
```

Comment:
we don't have to rename this all over, IMO "allowNavigation" is a lot more ambiguous about what is actually going to happen than "enableHistory" ... the *only* place we actually need to look at "allowNavigation" can be when we are taking the data. and we can leave it as legacy support with something like `enableHistory = page.custom.enableHistory || page.custom.allowNavigation` (checking for undefined and all that)